driver_author_analysis.py

```diff
@@ -1,13 +1,14 @@
-from lib.analysis.author.curve_fitting import generate_crt_curve_fits
-from lib.analysis.author.ranking import generate_author_ranking
-from lib.analysis.author.time_statistics import conversation_refresh_times
-from lib.analysis.author.wh_table import generate_wh_table_authors
-from lib.analysis.thread.hypergraph import generate_hyperedge_distribution
-from lib.input.mbox.keyword_clustering import generate_kmeans_clustering
-from lib.input.mbox.keyword_digest import generate_keyword_digest
+# from lib.analysis.author.curve_fitting import generate_crt_curve_fits
```

Comment:
The code for various analysis is not commented out in driver. Hence these imports may also need to remain?

Reply:
I had to do this because I did not have all the dependencies installed when testing the new script. Fixed in dab714efba09da9ca80442ccf2f4be5a6160bae4